ch_backup/clickhouse/control.py

```diff
@@ -757,6 +763,7 @@ def scan_frozen_parts(
                 checksum,
                 size,
                 rel_paths,
+                backup_meta.encrypted,
```

Comment:
@aalexfvk As far as I understand here should be used encrypted field from certain data part, but it is not clear for me, how to get part metadata, could you please suggest some variants? (I have tried to get it from backup_meta using find_part and get_parts functions, but results of these functions were empty)

Reply:
I think we should not add `encrypted` field to `FrozenPart` at all, because frozen parts on disk are never encrypted. Where do you use